blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -575,11 +575,21 @@ def test_interactive_len(sql):
     assert len(t) == int(t.count())
 
 
-def test_sample(sql):
-    t = symbol('t', discover(sql))
-    result = compute(t.sample(n=1), sql)
+def test_sample_n(nyc):
+    t = symbol('t', discover(nyc))
+    result = compute(t.sample(n=14), nyc)
     s = odo(result, pd.DataFrame)
-    assert len(s) == 1
-    result2 = compute(t.sample(frac=0.5), sql)
+    assert len(s) == 14
```

Comment:
Not sure I understand - could you explain a bit more please.

The `sample()` method will by default do a random sampling of the data in the SQL query by invoking `ORDER BY random()`.

In both cases we're doing a random sampling; however when using  `n=100` we have LIMIT defined as 100 while for `frac=0.5`, the LIMIT is computed as `COUNT(*) * 0.5`